src/helpers/tanStackQuery.tsx

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios'
```

Comment:
I suggest moving code from tanStackQuery, tanStackQueryColors and tanStackQuerySingle into single file (ex.: requests.ts, or something similiar) and also moving it from helpers folder to new "api" folder.